queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py

```diff
@@ -23,9 +23,14 @@
 from entity_queue_common.service_utils import logger
 from flask import current_app
 from jinja2 import Template
-from legal_api.models import Business, Filing
+from legal_api.models import Business, Filing, UserRoles
 
-from entity_emailer.email_processors import get_filing_info, get_recipients, substitute_template_parts
+from entity_emailer.email_processors import (
+    get_filing_info,
+    get_recipients,
+    get_user_email_from_auth,
+    substitute_template_parts,
+)
```

Comment:
Why the difference in styles in the imports?

Reply:
We have a max line length of 120